docs/source/quickstart/local-subgraphs.mdx

```diff
@@ -66,11 +74,11 @@ const server = new ApolloServer({
 
 (As shown, you now pass your `typeDefs` and `resolvers` to `buildSubgraphSchema`.)
 
-And that's it! You can now perform all of the same subgraph setup on your _own_ service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
+And that's it! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
```

Comment:
I like the exclamation point for the congratulatory sentence, but I don't like `And that's it`. Maybe just `Congratulations`? 

Or should we avoid congratulatory sentences?

```suggestion
Congratulations! You can now perform all of the same subgraph setup on your own service (schema publishing, introspection, etc.) that you did with the Apollo-hosted services we used in the previous parts. Refer to those parts for guidance.
```

Reply:
I think some congratulations are alright, particularly at the end of lengthy quickstarts.